cicd/bin/check_junit_files

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+import logging
+import sys
+
+from glob import iglob
+from junitparser import JUnitXml
+
+if __name__ == "__main__":
+
+    def errors_or_failures_found(junit_report):
+        for test_suite in junit_report:
+            if test_suite.errors or test_suite.failures:
+                return True
+        return False
+
+
+    logging.basicConfig(level=logging.INFO)
+
+    if len(sys.argv) == 1:
+        raise SystemExit("you need to provide a JUnit reports glob expression")
+
+    glob_path = sys.argv[1]
+
+    report_paths = list(iglob(glob_path))
+
+    if report_paths:
+        junit_xml = JUnitXml()
+        for report_path in report_paths:
+            junit_xml += JUnitXml.fromfile(report_path)
+
+        if errors_or_failures_found(junit_xml):
+            sys.exit(1)
+    else:
+        print("No artifacts found")
+        # TODO: should this fail?
```

Comment:
yes :)

Reply:
Done - if the glob expression does not expand into any file it will exit with error